Entities/Special/CTF/CTF_Flag.as

```diff
@@ -135,18 +140,37 @@ void onRender(CSprite@ this)
 
 		if (returncount < return_time)
 		{
-			GUI::DrawProgressBar(Vec2f(pos2d.x - 30.0f, pos2d.y - 10.0f + wave), Vec2f(pos2d.x + 30.0f, pos2d.y + 5.0f + wave), 1.0f - float(returncount) / float(return_time));
+			Vec2f pos = pos2d + Vec2f(-30.0f, -20.0f + wave);
```

Comment:
When you throw the flag the bar is really stuttery (similar problem with the drill heat bar). I think you should be using the interpolated position.